VocaDbWeb/Views/Shared/React/Index.cshtml

```diff
@@ -56,7 +57,16 @@
 </head>
 <body class="vdb">
 	<div id="app"></div>
-	<script>window.vdb = @ToJS(new { Resources = new GlobalResources(this), Values = new GlobalValues(this) })</script>
+	<script>
+		window.vdb = @ToJS(new
+		{
+			Resources = new GlobalResources(this),
+			Values = new GlobalValues(this)
+			{
+				RequestToken = Antiforgery.GetAndStoreTokens(Context).RequestToken,

```

Comment:
`React/Index.cshtml` is the single page that will host the React app. The same token will be used multiple times, unless a full page refresh is triggered. See also [javascript - Single Page Application and CSRF Token - Stack Overflow](https://stackoverflow.com/questions/50159847/single-page-application-and-csrf-token).